mathesar_ui/src/systems/table-view/table-inspector/TableInspector.svelte

```diff
@@ -26,6 +27,23 @@
   ];
 
   let activeTab: TabItem;
+
+  const tabularData = getTabularDataStoreFromContext();
+  $: ({ selection } = $tabularData);
+  $: ({ selectedCells } = selection);
+
+  $: {
+    // Explicit dependency
+    $selectedCells;
+
+    if (selection.isAnyColumnCompletelySelected()) {
+      [, activeTab] = tabs;
```

Comment:
This is pretty unreadable to me but eslint forced me to do this. Rule: `prefer-destructuring`

Reply:
Yeah, I'd be open to disabling that rule. I don't really like it either.

Reply:
It would be nice to leave it enabled for objects, but to disable the rule for arrays.